### PR TITLE
chore: add initial zep template

### PR DESCRIPTION
**Pull request by Racer159**
This PR adds the initial proposal template to this repo (cc @zarf-dev/maintainers)

### CODE REVIEW

File: NNNN-zep-template/README.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+Consider the following stages when setting graduation criteria:
+- Alpha: Feature is behind a feature flag, basic tests in place.
+- Beta: Gather feedback from users, complete core features, add more tests.
+- GA: Prove real-world usage, complete rigorous testing, gather feedback.
+
+In general, features should wait at least two releases between Beta and GA to
+allow time for feedback. For features moving to GA, include conformance tests
+to ensure stability and compatibility.
+
+#### Deprecation
```
Comment (AustinAbro321) [inline NNNN-zep-template/README.md:207]: @salaxander @schristoff Do we want this? We don't really have a deprecation process

File: .github/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+### Enhancement Description
+
+- One-line enhancement description (can be used as a release note):
+- Zarf Enhancement Proposal: <!-- link to zarf-dev/proposals file; if none yet, link to PR -->
+- Primary contact (assignee):
+- Enhancement target (which target equals to which milestone/version):
+  - Alpha release target (x.y):
+  - Beta release target (x.y):
+  - Stable release target (x.y):
```
Comment (AustinAbro321) [inline .github/issue_template.md:9]: @salaxander @schristoff Are we planning to put features through alpha stable and beta like Kubernetes?

**Review by AustinAbro321 — COMMENTED** [review]
Overall I think this looks good, thanks again for making it. 

Added a few question to the team around versioning since we are pretty different from Kubernetes in that aspect.

File: NNNN-zep-template/README.md [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+- [Proposal](#proposal)
+  - [User Stories (Optional)](#user-stories-optional)
+    - [Story 1](#story-1)
+    - [Story 2](#story-2)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Test Plan](#test-plan)
+      - [Prerequisite testing updates](#prerequisite-testing-updates)
+      - [Unit tests](#unit-tests)
+      - [e2e tests](#e2e-tests)
```
Comment (AustinAbro321) [inline NNNN-zep-template/README.md:74]: Could you add the corresponding headers for the sub points of Test Plan

**Review by AustinAbro321 — CHANGES_REQUESTED** [review]

**Review by schristoff — APPROVED** [review]

File: .github/issue_template.md (hunk 2 above)
Comment (schristoff) [reply .github/issue_template.md:9]: Yes

**Review by AustinAbro321 — APPROVED** [review]

**Pull request merged**